wordpress-mobile/WordPress-iOS
# Themes searching, filtering, customization

Closes #3903; closes #3904; closes #3905 

Adds searching, filtering, and current theme display customization/management functions to themes browser following design [here](https://teamhyperion.wordpress.com/2015/05/07/calypso-theme-showcase-in-calypso-basic-master-thread/).

To test: 
1. Enable themes feature with `wpdebug://themes?enabled=1` as discussed in #3913
2. Activate themes browser from the blog details screen
3. Note current theme loaded at list beginning, and display management functions for it available from top header bar. Check they load expected links.
4. Select free/premium/all themes and activate search from second header bar. Check that filtering acts as expected, including showing Drake when no results are available.

@jleandroperez Could you review some more theme stuff please?
Needs Review: @jleandroperez
